src/components/calcite-tip-group/calcite-tip-group.scss

```diff
@@ -1,12 +1,10 @@
-$tip-max-width: 540px;
 
 :host {
   @extend %component-host;
+  --tip-max-width: 540px;
```

Comment:
should this be scoped to `calcite`?

Reply:
Could do. Should already be scoped because it's inside `:host`, but if we wanna be super explicit, I'm cool with that.

Reply:
Updated.